components/dashboard/src/components/ProgressBar.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) 2023 Gitpod GmbH. All rights reserved.
+ * Licensed under the GNU Affero General Public License (AGPL).
+ * See License.AGPL.txt in the project root for license information.
+ */
+
+import { FC } from "react";
+
+type Props = {
+    percent: number;
+};
+export const ProgressBar: FC<Props> = ({ percent }) => {
+    return (
+        <div className="w-full rounded-full h-3 my-1 bg-gray-600 dark:bg-gray-300">
```

Comment:
**suggestion**: What do you think of using the same progress bar height as in `/billing`? Using `h-2` with some x-padding could help maintain the line height if needed.

| Billing Settings | Usage Export |
|-|-|
| <img width="608" alt="Frame 1563" src="https://github.com/gitpod-io/gitpod/assets/120486/0d9801d0-4545-4c50-9f5d-b24baaa9e9bf"> | <img width="608" alt="Frame 1564" src="https://github.com/gitpod-io/gitpod/assets/120486/820535b7-8146-4e9d-bc01-f3c5e15f3747"> |